atuin-client/config.toml

```diff
@@ -10,29 +10,46 @@
 # session_path = "~/.key"
 
 ## date format used, either "us" or "uk"
-# dialect = "uk"
+# dialect = "us"
```

Comment:
I believe the US is the only country that uses the US format of mm/dd/yy, vs most of the rest of the world in using dd/mm/yy

```suggestion
# dialect = "uk"
```